examples/jsm/controls/TrackballControls.js

```diff
@@ -706,6 +706,12 @@ class TrackballControls extends EventDispatcher {
 					_movePrev.copy( _moveCurr );
 					break;
 
+				case 2:
```

Comment:
Can you please describe in more detail why you have added this section? 

AFAICS, this is only relevant when three pointers were active and one of them was released. However, `TrackballControls` does not make use of more than two pointers. So what does this code exactly do?

Reply:
In my actual use that I found ,`case 2` will be triggered when I take away the one of tow pointers;
And` case 0` never be triggered

Reply:
this is my step on mobile
Step one: I zoom objcet with two pointer
Step two: I release one of them pointer
Step three:  I move left pointer then the object will rotate just like one pointer rorate  before but not zoom
I think it shouldn't be like that

Reply:
> Step one: I zoom objcet with two pointer
Step two: I release one of them pointer
Step three: I move left pointer then the object will rotate just like one pointer rorate before but not zoom

Um, but this sounds like the expected behavior. The state of the controls corresponds to the number of active pointers.

Reply:
But in step three I just move a little distance but the object rotate very much!

Reply:
And I continue move pointer , the object rotate will be normal,
It just happend in zoom finished once that rotate very much 
Buy if you release tow pointers same time ,it won't happen

Reply:
reference  #7406
but it's not fix this bug complete

Reply:
> And I continue move pointer , the object rotate will be normal,
It just happend in zoom finished once that rotate very much
Buy if you release tow pointers same time ,it won't happen

That cleared things up. Indeed, the "jump" does not appear with your version.

Reply:
Thanks man，so my pr will be merge？